Entry for 1-on-1 meetings (#35)
seedcase-project/team: entries/1-on-1-meetings.md

space to talk about needs for working and growing in a team environment
that is safe and supportive.

To help achieve this aim, here are a few things to keep in mind as some
expectations for these meetings:

-   The focus is on team member, how they're doing, and their needs in
    relation to the job.
-   We want to create a space that is as non-judgmental and safe (and
    confidential) as possible, so that we can discuss topics openly and
    without consequences.
-   While there will be a general agenda, it is mostly informal.

Some questions we might want to discuss or cover are:

-   How can the team lead or the team structure how we work or do things
    to better support you in your work?
-   What can the team lead or the team do to make you feel safe,
    supported, and secure in doing your work?
-   Do you feel like you are growing and learning the things you want to
    learn from this job? What can the team lead or the team do to help
    you in that?
-   Are there any personal or work related things that are blocking you
    or making it more difficult for you to do your tasks effectively?
-   What are things you want to learn or gain from this job for the next

>     check in? What can I or we do to help with that?

Author: ```suggestion
    check in? What can the team lead or the team do to help with that?
```
